.github/workflows/codeql.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "codeQlAct"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ,"codeQlAct"]
+  schedule:
+    - cron: '45 7 * * 0'
```

Comment:
Can you translate that cron to something human readable as comment for convenience ?